src/ImageSharp/ImageSharp.csproj

```diff
@@ -44,6 +44,9 @@
     <PackageReference Include="System.Memory" Version="4.5.0" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.0" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' != 'netstandard1.1' ">
```

Comment:
you can skip this for our net standard 2.0 > targets too so you probably want to change this to `'$(TargetFramework)' == 'netstandard1.3'` instead